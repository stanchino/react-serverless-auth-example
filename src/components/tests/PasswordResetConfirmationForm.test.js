import React from "react";
import { PasswordResetConfirmForm } from "..";
import { testContainer } from "./shared-examples";

describe("PasswordResetConfirmForm", () => testContainer(<PasswordResetConfirmForm />));