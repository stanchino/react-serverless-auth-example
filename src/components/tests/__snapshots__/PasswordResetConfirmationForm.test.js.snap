// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordResetConfirmForm renders without errors 1`] = `
<form
  className={undefined}
  onSubmit={[Function]}
>
  <div
    className="messages"
  >
    <div
      className="flash"
    />
  </div>
  <div>
    <label />
    <div>
      <input
        name="code"
        onBlur={[Function]}
        onChange={[Function]}
        onDragStart={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        placeholder="Confirmation Code"
        type="text"
        value=""
      />
    </div>
  </div>
  <div>
    <label />
    <div>
      <input
        autoComplete="new-password"
        name="password"
        onBlur={[Function]}
        onChange={[Function]}
        onDragStart={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        placeholder="Password"
        type="password"
        value=""
      />
    </div>
  </div>
  <div>
    <label />
    <div>
      <input
        autoComplete="new-password"
        name="password_confirmation"
        onBlur={[Function]}
        onChange={[Function]}
        onDragStart={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        placeholder="Confirm Password"
        type="password"
        value=""
      />
    </div>
  </div>
  <button
    disabled={true}
    type="submit"
  >
    Reset Password
  </button>
  <button
    disabled={true}
    onClick={[Function]}
    type="reset"
  >
    Cancel
  </button>
</form>
`;
